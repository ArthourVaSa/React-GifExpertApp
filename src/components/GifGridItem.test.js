import React from "react";
import {shallow} from "enzyme"

import { GifGridItem } from "./GifGridItem";

describe('Pruebas en el componente GifGridItem', () => {
    let wrapper = shallow(<GifGridItem/>);

    beforeEach(() => {
        wrapper = shallow(<GifGridItem/>);
    });

    test('Debe mostrar el GifGridItem', () => {
        const data = {
            url: "https://asdlaksjdalsd.com",
            title: "HolaMundo",
        }

        const wrapper = shallow(<GifGridItem {...data}/>);
        expect(wrapper).toMatchSnapshot();
    });

});