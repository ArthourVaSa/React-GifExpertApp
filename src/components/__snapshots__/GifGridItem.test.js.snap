// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente GifGridItem Debe mostrar el GifGridItem 1`] = `
<div
  className="card animate__animated animate__fadeInTopRight animate__delay-2s"
>
  <img
    alt="HolaMundo"
    src="https://asdlaksjdalsd.com"
  />
  <p>
    HolaMundo
  </p>
</div>
`;
